### PR TITLE
Consolidate WiW instantiations and sprinkle in es6

### PR DESCRIPTION
**Pull request by apackin**
#### What's this PR do?
Consolidate WiW instantiations and sprinkle in es6 const
#### How should this be manually tested?
You can run the tasks manually to make sure they still work via:
node console ${task file name} ${exported function name}

You can also npm test and run selected jobs.
#### What are the relevant tickets?
https://admin.crisistextline.org/jira/secure/RapidBoard.jspa?rapidView=5&view=detail&selectedIssue=INT-209

### CODE REVIEW

File: jobs/scheduling/RecurShifts.js [hunk 1]
```diff
@@ -207,7 +208,7 @@ function recurNewlyCreatedShifts(optionalNoShiftsForTesting) {
         }
       }
       async.waterfall(requestTaskArray, function(err, startDate, endDate, unpublishedShiftIDs) {
```
Comment (mbogor) [inline jobs/scheduling/RecurShifts.js]: i c u w the async waterfall

**Comment by mbogor** [comment]
👍

**Comment by apackin** [comment]
Thanks Mariya.
@tongxiang

File: api_wiw/WiWSuperApi.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+const WhenIWork = require('wheniwork-unofficial');
```
Comment (tongxiang) [inline api_wiw/WiWSuperApi.js]: why did you not include a 'use strict' statement here?

File: api_wiw/WiWCCApi.js [hunk 3]
```diff
@@ -0,0 +1,4 @@
+const WhenIWork = require('wheniwork-unofficial');
```
Comment (tongxiang) [inline api_wiw/WiWCCApi.js]: why did you not include a 'use strict' statement here?

File: api_wiw/WiWSuperApi.js (hunk 2 above)
Comment (apackin) [inline api_wiw/WiWSuperApi.js]: I didn't include it because no functions are being written. I can add it if you prefer.

Comment (tongxiang) [inline api_wiw/WiWSuperApi.js]: I feel like `use strict` governs variable assignment, too, in addition to functions. 

Your choice.

**Comment by tongxiang** [comment]
👍  good to merge! 

Good, clear thinking around re-structuring where the APIs get instantiated.

**Pull request merged**